### PR TITLE
bug fixes in Azure Webapp deploy task

### PR DESCRIPTION
**Pull request by vineetmimrot**
This PR addresses the following issues:
1. In case of publish profile scenario, change the slot name to "production", if the deployment is done on main webApp.

2. minor fix for printing application url on console log.

### CODE REVIEW

**Comment by vineetmimrot** [comment]
@bishal-pdMSFT can you review this PR.

**Comment by bishal-pdMSFT** [comment]
@vineetmimrot can you please add some details about the bug?

**Comment by vineetmimrot** [comment]
> @vineetmimrot can you please add some details about the bug?

Done. Can you please approve it.

File: Tasks/AzureWebAppV1/deploymentProvider/AzureRmWebAppDeploymentProvider.ts [hunk 1]
```diff
@@ -39,9 +39,7 @@ export class AzureRmWebAppDeploymentProvider implements IWebAppDeploymentProvide
             this.kuduService = new Kudu(this.publishProfileScmCredentials.scmUri, this.publishProfileScmCredentials.username, this.publishProfileScmCredentials.password);
             let resourceId = publishProfileEndpoint.resourceId;
             let resourceIdSplit = resourceId.split("/");
-            if (resourceIdSplit.length === 11) {
-                this.slotName = resourceIdSplit[10];
```
Comment (bishal-pdMSFT) [inline Tasks/AzureWebAppV1/deploymentProvider/AzureRmWebAppDeploymentProvider.ts]: can we not just use `this.taskParams.SlotName` for slotName

Comment (vineetmimrot) [reply Tasks/AzureWebAppV1/deploymentProvider/AzureRmWebAppDeploymentProvider.ts]: In the case of publish profile, the input fields related to webapp are redundant. The artifact will be deployed only at the scm url present in the publish profile, irrespective of what the user provides value into input fields.

Comment (bishal-pdMSFT) [reply Tasks/AzureWebAppV1/deploymentProvider/AzureRmWebAppDeploymentProvider.ts]: Lets take this out of this PR

**Pull request merged**